graphql_api/data_s3/base_s3_data.py

```diff
@@ -44,16 +45,23 @@ def get_one(self, _id):
         """
         raise NotImplementedError("method needs to be defined by sub-class")
 
-    def get_all(self):
+    def get_all(self, clazz_name=None):
```

Comment:
One of the tests mocks this function, which fails due to the updated function parameter:

`graphql_api\tests\test_schema.py:122: TypeError`
`{'errors': [{'message': '<lambda>() takes 1 positional argument but 2 were given', 'locations': [{'line': 2, 'column': 21}], 'path': ['rupture_generation_tasks']}], 'data': {'rupture_generation_tasks': None}}`

Would need to update the lambda used by the mock.

Reply:
Right - i have fixed this already, but on another branch! Ooops.